modules/virt-adding-secret-configmap-service-account-to-vm.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc
+
+[id="virt-adding-secret-configmap-service-account-to-vm_{context}"]
+
+= Adding a secret, ConfigMap, or service account to a virtual machine
+
+Add a secret, ConfigMap, or service account to a virtual machine by using the {product-title} web console.
+
+.Prerequisites
+
+* You must have at least one available secret, ConfigMap, or service account
+configured in your {product-title} cluster.
+
+* The virtual machine must be powered off.
+
+.Procedure
+
+. From the side menu, navigate to the *Virtualization* -> *Virtual Machines* page.
```

Comment:
As per ISG (347), this is probably better served by 'Click Virtualization -> ..'